File: Heavyweights/belzebuh/belzebuh.py

class Belzebuh:
    def __init__(self):
        self.form = "Demonic Giant"
        self.height = 4  # meters in lore
        self.on_map_height = "Standard"
        self.stamina_generator = True
        self.lane = "Highland"

    def rock_punch(self, hits_enemy=True):
        if hits_enemy:
            return "Damage and Slow"
        else:
            return "Knocks enemy upward"

    def headbutt(self):
        return "Lifts and throws enemy back, causing Damage and Armor Break"

    def gut_wrench(self):
        return "Back attack that causes Damage and Torpor"

    def hyperthrow(self):
        return "Grabs enemies, throws them upward and crushes them, dealing massive damage"

File: Heavyweights/belzebuh/Belzebuh.java

public class Belzebuh {
    private String form = "Demonic Giant";
    private int height = 4; // meters in lore
    private String onMapHeight = "Standard";
    private boolean staminaGenerator = true;
    private String lane = "Highland";

    public String rockPunch(boolean hitsEnemy) {
        if (hitsEnemy) {
            return "Damage and Slow";
        } else {
            return "Knocks enemy upward";
        }
    }

    public String headbutt() {
        return "Lifts and throws enemy back, causing Damage and Armor Break";
    }

    public String gutWrench() {
        return "Back attack that causes Damage and Torpor";
    }

    public String hyperthrow() {
        return "Grabs enemies, throws them upward and crushes them, dealing massive damage";
    }
}
